### DISCUSSION TITLE
No response from GitHub support

### DISCUSSION THREAD

**Discussion by amelnoah** [community/community#118622] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have been waiting for a response from GitHub support for 3 weeks in regards to my Reinstatement Request. It has affected my work as my colleagues were unable to see my profile and pull requests created by me.

Account Username: ameltatad
Ticket #2689058

Is there any other way to contact GitHub support or will I be stuck waiting for a fix?

**Comment by Luuan11** [comment]
Hey, How are you fine ? 

Probably you will need to reopen that ticket, Github support normally response early

**Comment by queenofcorgis** [comment, marked as answer]
Hi @amelnoah,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
